ui.frontend/src/GuideBridge.js

```diff
@@ -21,6 +21,7 @@ import AfFormData from "./FormData.js";
 export default class GuideBridge {
 
     #guideContainerMap = {};
+    #formContainerViewMap = {};
```

Comment:
lets leverage formContainerMap fo the use case of guidecontainermap also. and remove gyuidecontainermap.